__tests__/app.test.js

```diff
@@ -41,3 +44,27 @@ describe("GET API", () => {
       });
   });
 });
+
+describe("GET /api/article/:article_id", () => {
+  test("should return a 200 status code", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("should return an object with the correct article with correct keys", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((res) => {
+        expect(typeof res.body.article).toBe("object");
+        expect(typeof res.body.article[0].title).toBe("string");
+        expect(typeof res.body.article[0].topic).toBe("string");
+        expect(typeof res.body.article[0].author).toBe("string");
+        expect(typeof res.body.article[0].created_at).toBe("string");
+        expect(typeof res.body.article[0].votes).toBe("number");
+        expect(typeof res.body.article[0].article_img_url).toBe("string");
+        expect(typeof res.body.article[0].article_id).toBe("number");
+      });
+  });
+  test("should return 400 error when passed an article id that doesn't exist", () => {
+    return request(app).get("/api/articles/40").expect(400);
+  });
+});
```

Comment:
This should be a 404 - not a 400. It is a perfectly valid article_id, it just doesn't exist yet. 400 designates that this is a bad request, of which it isn't. 404 - Not Found, is the way we want to go